src/main/java/org/skife/jdbi/v2/sqlobject/SqlObjectBuilder.java

```diff
@@ -22,6 +22,7 @@
  */
 public class SqlObjectBuilder
 {
+    static final ParameterBinderRegistry  binderRegistry = new ParameterBinderRegistry();
```

Comment:
This being static makes me uncomfortable.

I'd rather this registry live in the DBI, and passed (possibly cloned) to each SqlObject.

We'll have to do something different for v3 since sqlobject is now a separate layer on top of core.